apps/condo/domains/common/hooks/useValidations.ts

```diff
@@ -70,13 +69,15 @@ export const useValidations: UseValidations = (settings = {}) => {
     const minLengthValidator: (length: number) => Rule = (length) => {
         return {
             min: length,
+            transform: (value) => value && value.trim(),
```

Comment:
Is it ok to change it here? Did you check, that this addition not breaking other use-cases?

Overall it looks ok :)

Reply:
It seems that without this addition, any cases where a length check is required do not work correctly, since most likely the stored value will be trimmed anyway